client/src/__tests__/inspect.test.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import { render, screen, waitFor, fireEvent } from '@testing-library/react';
+import userEvent from '@testing-library/user-event'
+import { navigate } from '@reach/router';
+import '@testing-library/jest-dom/extend-expect'
+import Inspect from '../pages/Inspect';
+import { deleteTransaction, getOneTransaction, getTransactions, editTransaction } from '../services/TransactionService';
+
+jest.mock('../services/TransactionService');
+getOneTransaction.mockResolvedValue('upload');
+getTransactions.mockResolvedValue('upload');
```

Comment:
I'm guessing you aren't using this mock resolved value, but perhaps don't use the word 'upload' for when you are doing a get request